include/dd/Package.hpp

```diff
@@ -333,6 +333,49 @@ namespace dd {
             return state;
         }
 
+        /**
+            Convert a matrix to a decision diagram.
+            This function converts a given matrix to a DD by iteratively breaking down the matrix
+            into quarters until we have single elements that are in the correct order for a decision diagram.
```

Comment:
that sentence might need a slight rewording now that the implementation has changed.

Reply:
Done